x/feegrant/client/cli/tx.go

```diff
@@ -125,25 +125,24 @@ Examples:
 					return err
 				}
 
-				if periodClock > 0 && periodLimit != nil {
```

Comment:
the check `periodClock > 0` is redundant as we are already checking that above on line 122

Reply:
Perodic-fee-allowance requires both flags, the above check is validating if the `period` or `period-limit` flag is set.

Reply:
👍 I have added another if check so that if period is not set we return an error reflecting that